openai-model-provider/proxy/validate.go

```diff
@@ -0,0 +1,74 @@
+package proxy
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"time"
+)
+
+func init() {
+	log.SetFlags(0)
+}
+
+func logInfo(loggerPath string, msg string) {
+	log.Printf("time=%q level=info msg=%q logger=%s", time.Now().Format(time.RFC3339), msg, loggerPath)
```

Comment:
Apart from the loggerPath this is not much different from the default log layout <time> <level> <msg> - why use a custom function here?
FWIW, `slog` is in the stdlib if you need structured logging (can e.g. enable json mode and use extra fields)

Reply:
fair. wasnt familiar with slog and was trying to match the servers log output format